content/en/borealis/quick-start/borealis-canary-get-started.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Get Started with Canary Analysis
+linktitle: Canary Analysis
+description: > 
+  This guide walks you through using canary analysis on the app you deployed in the Get Started with the CLI to Deploy Apps guide. Yu perform a retrospective analysis on the performance of the app that you can create canary analysis queries for subsequent deployments.s
```

Comment:
typo Yu -> You

Reply:
extra s in the end after period.

Reply:
"Yu perform a retrospective analysis on the performance of the app that you can create canary analysis queries for subsequent deployments"

the whole statement seems difficult to me, probably the "...that you can create canary..." phrasing is throwing me off.

Reply:
Haha wow. Reading it again, I think i was halfway through rewriting it and got distracted